controlplane/package.json

```diff
@@ -14,11 +14,13 @@
     "debug": "tsx watch src/index.ts",
     "build": "del dist && tsc",
     "seed": "API_KEY=cosmo_669b576aaadc10ee1ae81d9193425705 tsx src/bin/seed.ts",
+    "seed:billing": "tsx src/bin/seed-billing.ts",
     "db:migration-graph": "tsx --no-cache src/bin/create-request-tokens.ts",
     "db:generate": "drizzle-kit generate:pg --config=drizzle.config.ts",
     "db:custom": "drizzle-kit generate:pg --custom --config=drizzle.config.ts",
     "db:check": "drizzle-kit check:pg --config=drizzle.config.ts",
     "db:drop": "drizzle-kit drop --config=drizzle.config.ts",
+    "db:studio": "drizzle-kit studio --config=drizzle-studio.config.ts",
```

Comment:
Could you mention your flow in the readme?

Reply:
Actually I checked, and i was having issues with getting drizzle studio running, gonna drop it for now, but we can add it in a separate PR.